### ISSUE TITLE
Cancellation failed to create with "ErrorInvalidIdMalformed: The ID has an incorrect format"

### ISSUE THREAD

**Issue by talalong** [jamesiarmes/php-ews#407]
Version (e.g. 1.0, dev-master):
PHP version:
Microsoft Exchange version:
Exchange 2016
Description of problem:
I am not able to cancel events with ID and change_Key, that I have got from `event/create.php`
Please help!
Example request:

```
$event_id = 'AAMkADExNmU3YjU3LTBhOWEtNDQ5Ni05NGMzLWEwNGMwZWM4YTI4OQBGAAAAAADDOpI9BAzxQqjDD\/YLSAcjBwCjKmTAk1dzRLYkKY9vxZLUAAAAAAENAACjKmTAk1dzRLYkKY9vxZLUAAABVq2oAAA=';
$change_key = 'DwAAABQAAABylRmRhQetQb9WBnfOTsxMAAAwQg==';

// Set connection information.
$host = HOST;
$username = USER;
$password = PASS;
$version = Client::VERSION_2016;

$client = new Client($host, $username, $password, $version);
$client->setCurlOptions(array(CURLOPT_SSL_VERIFYPEER => false));
$request = new CreateItemType();
$request->MessageDisposition = MessageDispositionType::SEND_AND_SAVE_COPY;
$request->Items = new NonEmptyArrayOfAllItemsType();

$cancellation = new CancelCalendarItemType();
$cancellation->ReferenceItemId = new ItemIdType();
$cancellation->ReferenceItemId->Id = $event_id;
$cancellation->ReferenceItemId->ChangeKey = $change_key;
$request->Items->CancelCalendarItem[] = $cancellation;

$response = $client->CreateItem($request);

// Iterate over the results, printing any error messages.
$response_messages = $response->ResponseMessages->CreateItemResponseMessage;
foreach ($response_messages as $response_message) {
    // Make sure the request succeeded.
    if ($response_message->ResponseClass != ResponseClassType::SUCCESS) {
        $code = $response_message->ResponseCode;
        $message = $response_message->MessageText;
        var_dump("Cancellation failed to create with \"$code: $message\"\n");
        continue;
    }
}


```

Example response:

```
string(100) "Cancellation failed to create with "ErrorInvalidIdMalformed: The ID has an incorrect format." " 
```

Additional details:

**Comment by jamesiarmes** [comment]
Is the "\" character in your id supposed to be there? (i.e. was that actually in the id returned by Exchange). If so, you would need to escape it.

**Comment by talalong** [comment]
Yes, the "\" character should not be there. But all IDs I have got via ews contain that "\" character (just want to let you know, problem solved by escaping "\" from ID String ;)). 
By the way I can only cancel events that have guests who were invited in. Is there other function like `DeleteCalendarItem()`?
Because `CancelCalendarItem` could not delete or cancel my own events that have no guest.
I got this message:
Example response:
> At least one recipient is not valid., A message can not be sent because it does not contain recipients
